### DISCUSSION TITLE
Cannot read available PS2 savegames

### DISCUSSION THREAD

**Discussion by lcoser79** [aldostools/webMAN-MOD#506] [Q&A]
Dear all,

I've rebuilt my PS3 database and installed webMAN-MOD 1.47.35, but now I cannot read my PS2 savegames anymore, even though they are being properly shown in the PS3 embedded memory card manager. When starting the ISO using with webMAN the game simply tells me that it cannot find the memory card.

I've tried to search on Google, but unfortunatly it was a dead end. Any hints on what I could try?

Thanks in advance!

**Comment by aldostools** [comment]
Try assigning the memory card again through XMB before launch the game.

BTW This is not an issue of webMAN MOD.

**Comment by lcoser79** [comment]
Hi! Thank you very much for your help. That really did the trick! :-)

Kind regards and happy eastern!
